invenio_app_ils/ui/src/invenio_app_ils/common/api/urls.js

```diff
@@ -3,6 +3,7 @@ export const ApiURLS = {
     loan: '/circulation/loans/:loanPid',
     list: '/circulation/loans/',
     checkout: '/circulation/loans/checkout',
```

Comment:
could you please remove the checkout link because it is not valid, I thought I removed it maybe it's a rebase leftover?